packages/node-cli/src/ApplicationConstants.js

```diff
@@ -68,5 +68,23 @@ module.exports = {
 		SUITECLOUD_CI: 'SUITECLOUD_CI',
 		SUITECLOUD_CI_PASSKEY: 'SUITECLOUD_CI_PASSKEY',
 		SUITECLOUD_FALLBACK_PASSKEY: 'SUITECLOUD_FALLBACK_PASSKEY'
-	}
+	},
+
+	ACCOUNT_SETUP_CI: {
+		COMMAND: {
+			OPTIONS: {
+				ACCOUNT: 'account',
+				AUTHID: 'authid',
+				CERTIFCATEID: 'certificateid',
+				PRIVATEKEYPATH: 'privatekeypath',
+				DOMAIN: 'domain',
+				SELECT: 'select',
+			},
+			SDK_COMMAND: 'authenticateci',
+			OPTIONAL_IN_SETUP_MODE: ['domain'],
+		},
+		ACTION_RESULT: {
+			MODES: { REUSE: 'REUSE' },
+		},
```

Comment:
I would prefer to not add this constants here.
The values for ApplicationConstants are mostly for constants that are at "global" level. If we want to introduce some constants that are command specific, I would prefer at least to nest them into a `COMMANDS`  section.

My preference (in order) would be to go for one of the following approaches:

- expose static constants from `packages/node-cli/src/commands/account/setupci/AccountSetupCiAction.js`
- create `packages/node-cli/src/commands/CommandConstants.js` with all commands constants
- create `COMMANDS` nested property into `packages/node-cli/src/ApplicationConstants.js`
- create `packages/node-cli/src/commands/account/setupci/AccountSetupCiConstants.js`